compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -55,7 +55,7 @@
  */
 public class BLangParserListener extends BallerinaParserBaseListener {
     private static final String KEYWORD_PUBLIC = "public";
-    private static final String KEYWORD_NATIVE = "native";
+    private static final String KEYWORD_NATIVE = "extern";
```

Comment:
Shouldn't this be changed to `KEYWORD_EXTERN`? Just for the sake of consistency.